mast_aladin/app_sidecar.py

```diff
@@ -127,19 +149,55 @@ def open(
                 "run `pip install jdaviz`.",
                 UserWarning
             )
+        return apps
 
-        n_columns = len(apps)
+    def _attach_sidecars(self, apps, anchor, title):
```

Comment:
In a future update, let's auto-generate the title with the name of the app in that sidecar.